src/runtime-rs/crates/hypervisor/ch-config/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "ch-config"
```

Comment:
I think this package could be named as `ch-libs` because it not only includes config